src/FieldController.php

```diff
@@ -27,17 +27,22 @@ public function upload(Request $request)
 
             $data[] = [
                 'image' => $savedImage,
+                'saved_path' => $savedImage,
                 'url' => Storage::disk($disk)->url($savedImage),
             ];
         }
 
         return $data;
     }
 
-    public function delete($image)
+    public function delete(Request $request, $saved_path)
     {
-        Storage::delete($image);
+        $disk = $request->disk ? $request->disk : 'local';
+
+        $deleted = Storage::disk($disk)->delete($saved_path);
 
-        return "success";
+        return $deleted === true
+            ? 'success'
+            : response('not deleted', 404);
```

Comment:
I don't know if this should be really a RESTful controller, but this works so I don't know about no use.
Presently, there is not even a real entity being represented here, but I've been thinking that it could be a way to handle #13 having a handler of Preloaded images but not finally attached to the resource, like abandoned editing.

I haven't time to start a new package handling all these issues at once, and was expecting the original writer could attend to the PR's but I'll keep in mind your observation and think the handling of the images in a more restful way.